### DISCUSSION TITLE
change the number of letters in the title of days/months etc.

### DISCUSSION THREAD

**Discussion by AnatoliiCubex** [wojtekmaj/react-date-picker#506] [Q&A] [answered]
Hi there, is there any option to change the number of letters in the title of days/months etc.? For example I have days shown as:
**Mon Tue Wed Thu Fri Sat Sun**
But I want them to be shown as:
**Mo Tu We Th Fr Sa Su**
Same with monthes, January as Jan, February as Feb, etc.

**Comment by wojtekmaj** [comment, marked as answer]
Sure, this was added in https://github.com/wojtekmaj/react-calendar/issues/620 - you can pass your own formatting function to React-Date-Picker and it will get passed to React-Calendar.

**Comment by AnatoliiCubex** [comment]
Thank you, my bad I didnt check props for Calendar only, was looking in DatePicker section
